File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/controllers/linecharts/VisualsController.java
package com.kendoui.spring.controllers.linecharts;

import java.text.ParseException;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

import com.kendoui.spring.models.ChartDataRepository;

@Controller("dataviz-line_charts-visuals-controller")
@RequestMapping(value="/line-charts/")
public class VisualsController {
    @RequestMapping(value = "/visuals", method = RequestMethod.GET)
    public String index(Model model) throws ParseException {
        model.addAttribute("forecast", ChartDataRepository.ForecastData());
        
        return "/line-charts/visuals";
    }
}

File: wrappers/mvc/demos/Kendo.Mvc.Examples/Controllers/Line_Charts/VisualsController.cs
using System;
using System.Linq;
using System.Web.Mvc;
using Kendo.Mvc.Examples.Models;

namespace Kendo.Mvc.Examples.Controllers
{
    public partial class Line_ChartsController : Controller
    {
        [Demo]
        public ActionResult Visuals()
        {
            return View(ChartDataRepository.ForecastData());
        }
    }
}